perun-core/src/main/java/cz/metacentrum/perun/core/entry/VosManagerEntry.java

```diff
@@ -198,6 +221,19 @@ public Vo getVoById(PerunSession sess, int id) throws VoNotExistsException, Priv
 		return vo;
 	}
 
+	@Override
+	public EnrichedVo getEnrichedVoById(PerunSession sess, int id) throws VoNotExistsException, PrivilegeException {
+		Utils.notNull(sess, "sess");
+		EnrichedVo enrichedVo = vosManagerBl.getEnrichedVoById(sess, id);
+
+		// Authorization
+		if (!AuthzResolver.authorizedInternal(sess, "getEnrichedVoById_int_policy", enrichedVo.getVo())) {
+			throw new PrivilegeException(sess, "getEnrichedVoById");
+				}
```

Comment:
Please, fix this indentation.
```suggestion
		}
```